app/manifest.webapp

```diff
@@ -1,6 +1,6 @@
 {
   "name": "InternetRadio",
-  "description": "Internet Radio",
+  "description": "App to listen to Internet radios",
```

Comment:
Can you change radios to radio? I think it sounds more grammatically correct that way. Then I'd be glad to accept the pull request!